app/scripts/modules/kubernetes/src/v2/pipelines/stages/patchManifest/patchManifestStage.ts

```diff
@@ -0,0 +1,62 @@
+import { module } from 'angular';
+
+import {
+  ArtifactReferenceService,
+  ExecutionArtifactTab,
+  ExecutionDetailsTasks,
+  Registry,
+  SETTINGS,
+} from '@spinnaker/core';
+import { KubernetesV2PatchManifestConfigCtrl } from '../patchManifest/patchManifestConfig.controller';
+import { KUBERNETES_PATCH_MANIFEST_OPTIONS_FORM } from './patchOptionsForm.component';
+import { KUBERNETES_MANIFEST_SELECTOR } from '../../../manifest/selector/selector.component';
+import { DeployStatus } from '../deployManifest/react/DeployStatus';
+import { IPipeline, IStage } from '../../../../../../core/src/domain';
+import { ICustomValidator } from '../../../../../../core/src/pipeline/config/validation/PipelineConfigValidator';
+
+export const KUBERNETES_PATCH_MANIFEST_STAGE = 'spinnaker.kubernetes.v2.pipeline.stage.patchManifestStage';
+
+export class PatchStatus extends DeployStatus {
```

Comment:
I don't love this but it's fine for now.

Reply:
I'll separate this out into a common class that can be extended by both DeployStatus and PatchStatus.